internal/config/apparmor/apparmor_linux.go

```diff
@@ -23,7 +24,7 @@ type Config struct {
 // New creates a new default AppArmor configuration instance.
 func New() *Config {
 	return &Config{
-		enabled:        apparmor.IsEnabled(),
+		enabled:        runcaa.IsEnabled(),
```

Comment:
typically we call this libctr, so maybe libctraa?

Reply:
sure, fixed.